File: pyg-manager/pyg-manager-interface/src/main/java/com.pyg.manager.service/BrandService.java
package com.pyg.manager.service;

import com.pyg.pojo.TbBrand;
import com.pyg.utils.PageResult;

import java.util.List;
import java.util.Map;

/**
 * Created by on 2018/8/10.
 */
public interface BrandService {
    /**
     * 需求：查询所有品牌数据
     */
    public List<TbBrand> findAll();

    PageResult findAllByPage(int pageNum, int pageSize);

    public void save(TbBrand brand);

    public TbBrand findById(long id);

    public void update(TbBrand tbBrand);

    public void deleteById(long id);

    public List<Map> findBrandList();

    public PageResult findAllByPage(TbBrand tbBrand, int pageNum, int pageSize);
}

File: pyg-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService",function ($http) {
    this.findByPage=function (pageNum,pageSize) {
        return $http.get("../brand/findAllByPage?pageNum="+pageNum+"&pageSize="+pageSize);
    };
    this.findById=function (id) {
        return $http.get("../brand/findById?id="+id);
    };
    this.dele=function (ids) {
        return $http.get("../brand/delete?ids="+ids);
    };
    this.save=function (brand) {
        return $http.post("../brand/save",brand);
    };
    this.update=function (brand) {
        return $http.post("../brand/update",brand);
    }
    this.findBrandList=function () {
        return $http.get("../brand/findBrandList");
    }
    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post("../brand/search?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
    }
});